### PR TITLE
[rpm] Bump foreman_ansible_core to 2.0.2

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by dLobatog** [comment]
DO NOT MERGE YET - I've found in some tests that this version of foreman_ansible_core is NOT working well with the proxy

**Comment by dLobatog** [comment]
👍 now

File: rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec [hunk 1]
```diff
@@ -16,6 +16,8 @@ Requires: %{?scl_prefix_ruby}ruby
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(foreman-tasks-core) >= 0.1
 Requires: %{?scl_prefix}rubygem(foreman-tasks-core) < 1.0
+Requires: %{?scl_prefix}rubygem(foreman_remote_execution_core) >= 1.1
```
Comment (ares) [inline rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec]: there's no such rex core, rex core was not released as there was no change in its codebase

Comment (ares) [reply rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec]: please ignore me, the package is there, it's just not packaged for some reason

Comment (ares) [reply rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec]: it seems it was bumped by https://github.com/theforeman/foreman-packaging/commit/2028d1d18eee68c471c1e9eafdd082ec11cd8a23 but never built, could someone who merges this also trigger the build pls?

Comment (ares) [reply rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec]: resolved with @ekohl package is built now

**Pull request merged**